lib/cmds/init/success.ts

```diff
@@ -0,0 +1,68 @@
+import { highlight } from 'cli-highlight'
+
+export type ConnectionType = 'Contentful.js' | 'GraphQL' | 'REST API'
+
+const FEEDBACK_LINK = 'https://contentful.typeform.com/contentful-cli'
+
+export default async function success({
+  accessToken,
+  connectionType,
+  entryId = '{entry_id}',
+  environmentId = '{environment_id}',
+  spaceId = '{space_id}'
+}: {
+  accessToken: string
+  connectionType: ConnectionType
+  entryId?: string
+  environmentId?: string
+  spaceId?: string
+}) {
+  const examples = {
+    'Contentful.js': highlight(
+      `
+  const contentful = require("contentful");
+  const client = contentful.createClient({
+    space: "${spaceId}",
+    accessToken: "${accessToken}"
+  });
+  client
+    .getEntry("${entryId}")
+    .then(entry => console.log(entry))
+    .catch(err => console.log(err));
+    `
+    ),
```

Comment:
second parameter accepts `language: 'javascript'`, it should help with coloring these 

```js
highlight(graphQlExample, {
  language: "JavaScript",
})
```